jdbc-adapter/pom.xml

```diff
@@ -60,10 +60,14 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.2</version>
-                <configuration>
-                    <source>${java.version}</source>
-                    <target>${java.version}</target>
-                </configuration>
+
+<configuration>
+					<source>1.7</source>
+					<target>1.7</target>
+					<fork>true</fork>
+					<executable>C:\Program Files\Java\jdk1.8.0_102\bin\javac</executable>
+  <encoding>UTF8</encoding>
+				</configuration>
```

Comment:
I think this causes an error of the build in Travis (the path is only valid for your system), we should change this whole section back. Maybe we have to adapt the PATH environment variable on your system so that building works on your system.